assets/app/views/directives/delete-button.html

```diff
@@ -1,7 +1,10 @@
 <div class="actions action-button">
-	<i class="fa fa-trash-o" 
-	  ng-click="$event.stopPropagation(); openDeleteModal();"
-	  data-toggle="tooltip" 
-	  data-placement="left" 
-	  title="Delete"></i>
-</div>
\ No newline at end of file
+  <i class="fa fa-trash-o"
+    ng-click="$event.stopPropagation(); openDeleteModal();"
+    data-toggle="tooltip"
+    data-placement="left"
+    disable-popup-on-touch="true"
+    title="Delete"
+    aria-hidden="true"></i>
+  <a class="sr-only" href="" ng-click="openDeleteModal()">Delete {{resourceType | humanizeResourceType}} {{resourceName}}</a>
```

Comment:
Yeah that's fine.  And if we remove the hover popup then we can use the
regular title attribute for the sr text.

On Tue, Dec 1, 2015 at 3:33 PM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/directives/delete-button.html
> <https://github.com/openshift/origin/pull/6152#discussion_r46334568>:
>
> > \ No newline at end of file
> > +  <i class="fa fa-trash-o"
> > +    ng-click="$event.stopPropagation(); openDeleteModal();"
> > +    data-toggle="tooltip"
> > +    data-placement="left"
> > +    disable-popup-on-touch="true"
> > +    title="Delete"
> > +    aria-hidden="true"></i>
> > +  <a class="sr-only" href="" ng-click="openDeleteModal()">Delete {{resourceType | humanizeResourceType}} {{resourceName}}</a>
>
> Need to add the action-button class directly to the link or it turns the
> icon blue.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6152/files#r46334568>.
>